src/api/providers/fetchers/modelCache.ts

```diff
@@ -76,17 +65,8 @@ export const getModels = async (options: GetModelsOptions): Promise<ModelRecord>
 		}
 
 		// Cache the fetched models (even if empty, to signify a successful fetch with no models)
-		memoryCache.set(provider, models)
-		await writeModels(provider, models).catch((err) =>
-			console.error(`[getModels] Error writing ${provider} models to file cache:`, err),
-		)
+		memoryCache.set(cacheKey, models)
 
-		try {
-			models = await readModels(provider)
```

Comment:
This cache file seems to be ineffective, because it is only read after we just wrote it, so I decided to remove it.